Library/OcDebugLogLib/DebugPrint.c

```diff
@@ -36,6 +36,23 @@ DebugPrintDevicePath (
   DEBUG_CODE_END ();
 }
 
+VOID
+DebugPrintDevicePathForHandle (
+  IN UINTN                     ErrorLevel,
+  IN CONST CHAR8               *Message,
+  IN EFI_HANDLE                Handle
+  )
+{
+  DEBUG_CODE_BEGIN ();
+  
+  EFI_DEVICE_PATH_PROTOCOL  *DevicePath;
+
+  DevicePath = DevicePathFromHandle (Handle);
+  DebugPrintDevicePath (ErrorLevel, Message, DevicePath);
+
+  DEBUG_CODE_END ();
+}
+
```

Comment:
Also just to double-check this is okay to add?

Reply:
DebugPrintDevicePath does not take OPTIONAL DevicePath, so better to check the result of DevicePathFromHandle for NULL. This change will need to be added to all Debug libraries in OpenCore.

Reply:
I've updated and added new question on revised commit.